stock_release_channel_block_backorder/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+Block delivery of unavailable products on backorders.
+
+A blocked delivey cannot be assigned to a release channel.
+To unblock such delivery a new order has to be created, or the user has to
```

Comment:
Should be noted that you likely need the stock_picking_group_by_partner_by_carrier/ module

Reply:
added